kubos/init.py

```diff
@@ -20,14 +20,17 @@
 import shutil
 import sys
 
-from kubos.utils.git_common import KUBOS_EXAMPLE_DIR
+from kubos.utils.git_common import KUBOS_RT_EXAMPLE_DIR
+from kubos.utils.git_common import KUBOS_LINUX_EXAMPLE_DIR
 from yotta import link, link_target
 from yotta.lib import folders
 from yotta.lib.detect import systemDefaultTarget
 
 def addOptions(parser):
     parser.add_argument('proj_name', nargs=1, help='specify the project name')
-
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('-l', '--linux', action='store_true', help='Initialize Kubos SDK project for KubOS Linux')
```

Comment:
It will.  From the Python docs: "'store_true' and 'store_false' - These are special cases of 'store_const' using for storing the values True and False respectively. In addition, they create default values of False and True respectively."